subsys/net/l2/wifi/CMakeLists.txt

```diff
@@ -23,65 +23,65 @@ if (CONFIG_WIFI_NM)
 zephyr_iterable_section(NAME wifi_nm_instance GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN ${CONFIG_LINKER_ITERABLE_SUBALIGN})
 endif()
 
-# Wi-Fi Enterprise test certificates handling
-set(gen_inc_dir ${ZEPHYR_BINARY_DIR}/misc/generated)
-set(gen_dir ${gen_inc_dir}/wifi_enterprise_test_certs)
-# Create output directory for test certs
-file(MAKE_DIRECTORY ${gen_dir})
+if(CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE)
```

Comment:
can you please also add `if CONFIG_NET_L2_WIFI_SHELL` as these are only used by Wi-Fi shell?

Reply:
Done.